src/transit_equity/networks/network_prep.py

```diff
@@ -1,38 +1,162 @@
+    gtfs_table : str
+        The name of the GTFS stops table.
+    user_type : str
+        The passenger type ID used to filter the data. In the orca_ng table, the groups are as
+        follows:
+            1 = Adult
+            2 = Youth
+            3 = Senior
+            4 = Disabled
+            5 = Low Income
+        Note: This may need to be updated if the codes were changed between the new ORCA db and 
+        orca_ng. I think that the new ORCA db uses letters to signify the card types, and that the 
+        column name that stores the passenger types may be named differently. Double check if this 
+        is true when using the updated database.
+    
+    Returns
+    -------
+    GeoDataFrame
+        A GeoDataFrame containing the trip data, with geometries set to 'board_location_shapely'.
+    """
 
-"""
+    # Check database urls are correct
+    print(os.getenv(postgres_url_ng))
 
-import pandas as pd
-import geopandas as gpd
-from transit_equity.geospatial.format_conversions import load_wkb
+    #connect to engines
+    engine_ng = create_engine(os.getenv(postgres_url_ng))
+
+    # Setup Session Maker and Session
+    session_ng_maker = sessionmaker(bind=engine_ng)
+    session_ng = session_ng_maker()
+
+    # NG test Schema Base
+    base_ng_test = get_automap_base_with_views(engine=engine_ng, schema=test_schema)
+    base_ng_orca = get_automap_base_with_views(engine=engine_ng, schema=orca_schema)
+
+    # Tables of interest from orca_ng
+    trips_ng = base_ng_test.metadata.tables[trips_table]
+    alights_ng = base_ng_test.metadata.tables[alights_table]
+    boardings_ng = base_ng_test.metadata.tables[boardings_table]
+    vboardings_ng = base_ng_orca.metadata.tables[vboardings_table]
+    gtfs_stops_ng = base_ng_test.metadata.tables[gtfs_table]
+
+    # Constructing the query
+    query = (
+        session_ng.query(
+            vboardings_ng.c.card_id,
+            vboardings_ng.c.txn_id,
+            alights_ng.c.txn_id,
+            vboardings_ng.c.device_dtm_pacific,
+            alights_ng.c.alight_dtm_pacific,
+            boardings_ng.c.stop_location,
+            gtfs_stops_ng.c.stop_location
+        ).select_from(trips_ng)
+        .join(boardings_ng, boardings_ng.c.txn_id == trips_ng.c.orig_txn_id)
+        .join(alights_ng, alights_ng.c.txn_id == trips_ng.c.dest_txn_id)
+        .join(vboardings_ng, vboardings_ng.c.txn_id == trips_ng.c.orig_txn_id)
+        .join(gtfs_stops_ng, gtfs_stops_ng.c.stop_id == alights_ng.c.stop_id)
+        .filter(
+            and_(
+                boardings_ng.c.stop_location.isnot(None),
+                gtfs_stops_ng.c.stop_location.isnot(None),
+                vboardings_ng.c.passenger_type_id == user_type
+            )
+        )
+    )
+
+    # Because the adults table is so large that it was causing memory limitation issues, read the
+    # table in chunks of 100000 rows and then concatenate them after.
+    chunk_size = 100000
```

Comment:
It would be great if we could parameterize the chunk_size for more flexibility.

Reply:
We could add another argument/parameter to this function called chunk size. 
Please do give it a default value of whatever you feel is appropriate. (You can look into some of my functions for examples of default arguments)